apps/meteor/app/lib/server/functions/saveUser/saveUser.ts

```diff
@@ -158,6 +173,16 @@ const _saveUser = (session?: ClientSession) =>
 		await Users.updateFromUpdater({ _id: userData._id }, updater, { session });
 
 		await onceTransactionCommitedSuccessfully(async () => {
+			if (session && options?.auditStore) {
+				// setting this inside here to avoid moving `executeSetUserActiveStatus` from the endpoint fn
+				// updater will be commited by this point, so it won't affect the external user activation/deactivation
+				if (userData.active) {
```

Comment:
```suggestion
				if (userData.active !== undefined) {
```
Should log deactivations too, no?